scribe.perl

```diff
@@ -172,7 +174,7 @@
 my $mathjax =			# undef = no math; string is MathJax URL
   'https://www.w3.org/scripts/MathJax/3/es5/mml-chtml.js';
 my $islide =			# String is i-slide library URL
-  'https://w3c.github.io/i-slide/i-slide-2.js?selector=a.islide';
+  'https://w3c.github.io/i-slide/i-slide.js?selector=a.islide';
```

Comment:
```suggestion
  'https://w3c.github.io/i-slide/i-slide-2.js?selector=a.islide';
```
I think we should keep the versioned link for the i-slide library, otherwise old minutes might break of a backwards incompatible change gets brought to it.